sdk/spring/README.md

```diff
@@ -1,391 +1,110 @@
-spring-starter-azure-cache | [![Maven Central][azure-spring-cloud-starter-cache-mvn-1.2.x-img]][azure-spring-cloud-starter-cache-mvn-1.2.x]
-spring-cloud-azure-eventhubs-stream-binder  | [![Maven Central][spring-cloud-azure-eventhubs-stream-binder-mvn-1.2.x-img]][spring-cloud-azure-eventhubs-stream-binder-mvn-1.2.x]
-spring-cloud-starter-azure-eventhubs | [![Maven Central][spring-cloud-starter-azure-eventhubs-mvn-1.2.x-img]][spring-cloud-starter-azure-eventhubs-mvn-1.2.x]
-spring-cloud-starter-azure-eventhubs-kafka | [![Maven Central][spring-cloud-starter-azure-eventhubs-kafka-mvn-1.2.x-img]][spring-cloud-starter-azure-eventhubs-kafka-mvn-1.2.x]
-spring-cloud-azure-servicebus-queue-stream-binder | [![Maven Central][spring-cloud-azure-servicebus-queue-stream-binder-mvn-1.2.x-img]][spring-cloud-azure-servicebus-queue-stream-binder-mvn-1.2.x]
-spring-cloud-azure-servicebus-topic-stream-binder | [![Maven Central][spring-cloud-azure-servicebus-topic-stream-binder-mvn-1.2.x-img]][spring-cloud-azure-servicebus-topic-stream-binder-mvn-1.2.x]
-spring-cloud-starter-azure-servicebus | [![Maven Central][spring-cloud-starter-azure-servicebus-mvn-1.2.x-img]][spring-cloud-starter-azure-servicebus-mvn-1.2.x]
-spring-cloud-starter-azure-storage-queue | [![Maven Central][spring-cloud-starter-azure-storage-queue-mvn-1.2.x-img]][spring-cloud-starter-azure-storage-queue-mvn-1.2.x]
-spring-starter-azure-storage **(Deprecated)** | [![Maven Central][azure-spring-starter-storage-mvn-1.2.x-img]][azure-spring-starter-storage-mvn-1.2.x]
+[Spring Cloud](https://spring.io/projects/spring-cloud) provides boilerplate patterns for developers to quickly build and orchestrate their microservice based applications. Based on that, **Spring Cloud for Azure** is designed to provide seamless Spring integration with Azure services. Developers can adopt a Spring-idiomatic way to take advantage of services on Azure, with only few lines of configuration and minimal code changes.
+
+#### Artifact IDs
+
+| Artifact ID                                       |
+| ------------------------------------------------- |
+| azure-spring-cloud-starter-cache                  |
+| azure-spring-cloud-starter-eventhubs              |
+| azure-spring-cloud-starter-eventhubs-kafka        |
+| azure-spring-cloud-starter-servicebus             |
+| azure-spring-cloud-starter-storage-queue          |
+| azure-spring-cloud-stream-binder-eventhubs        |
+| azure-spring-cloud-stream-binder-servicebus-queue |
+| azure-spring-cloud-stream-binder-servicebus-topic |
+| azure-spring-integration-eventhubs                |
+| azure-spring-integration-servicebus               |
+| azure-spring-integration-storage-queue            |
+
+#### Choose Version According to Spring Boot Version
+
+| Spring Boot Version | Spring Cloud for Azure    |
+| ------------------- | ------------------------- |
+| 2.5.x               | 2.12.0                    |
+| 2.6.x               | 2.12.0                    |
```

Comment:
Add the Spring Cloud version too here?

Reply:
I think `Spring Boot`'s version is the single source of truth.
Both `Spring Cloud` and `Spring Cloud for Azure`'s version should compatible with `Spring Boot`'s version.

It's customer's responsibility to make sure `Spring Cloud`'s version is compatible with `Spring Boot`'s version.
So I added a commit about `Version Mapping Between Spring Cloud And Spring Boot`.